Inserting split into boss attacks to separate boss attacks and user attacks on two different lines.

Hus274 opened the pull request:
fixes #3453 
### Changes

Splits the markdown text notification to the party into two code tags split by a newline instead of one code tag. The code tag prevents wrap-text css styles from being applied which was the cause of the scrolling.

The newline can be removed so that it only splits into two lines if needed by the browser size which might be slightly more ideal as Markdown doesn't recommend counting on line break escapes. This would be easy to change by simply updating the '  \n' to a comma or semicolon but the newline seem to be the preferred solution in the issue.

---

UUID: 2a4d30ee-4833-420e-be7c-ad6a6e261c5f

Related issue: https://github.com/HabitRPG/habitrpg/issues/3453

Hus274 committed:
Inserting split into boss attacks to seperate boss attacks and user attacks on two different lines.

Related issue: https://github.com/HabitRPG/habitrpg/issues/3453

Hus274 committed:
Inserting split into boss attacks to seperate boss attacks and user attacks on two different lines.

Related issue: https://github.com/HabitRPG/habitrpg/issues/3453

crookedneighbor commented:
Seems reasonable to me.

Alys commented:
I like the idea of not using the newline, but a comma or semicolon would look out of place. Is a space enough to make it split when the browser window is small?

Hus274 commented:
Yep, as long as they're in two different code blocks, it should automatically wrap the text to a new line.

Hus274 committed:
Merge branch 'develop' of https://github.com/Hus274/habitrpg into develop

Hus274 commented:
Changed to a space instead of a newline addressing Alys's comment above and updated the remote branch to avoid annoying merge commit necessity. Doing this resulting in this pull request closing. New pull request with updated code can be found here: https://github.com/HabitRPG/habitrpg/pull/7013
